ajax/libs/jquery.loadtemplate/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.loadtemplate",
+  "description": "jQuery plugin for loading and using templates. The plugin is designed to be simple yet powerful, and supports loading templates from within the page, or using AJAX to load html files.",
+  "filename": "jquery.loadTemplate.min.js",
+  "keywords": [
+    "templates",
+    "templating",
+    "jquery-plugin"
+  ],
+  "author": "Paul Burgess",
+  "homepage": "https://codepb.github.io/jquery-template/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/codepb/jquery-template.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/codepb/jquery-template.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(*[0-9])*"
```

Comment:
@extend1994 Should we help to change the rule as `**/*`? 

Because we add the old versions and we use this pattern under `dist` usually.
As the past, we use `**/*` but manually rename filenames in some versions to the correct naming scheme.

Thank you.
(Sorry for my poor English.)

Reply:
Should I make a change?

Reply:
@sufuf3 I didn't change it because I am afraid to see the file naming scheme changes again. But if you think it would better to use `**/*`. It is fine to me to make this change.
@sashberd Thanks for quick attention. Just discussion here. Let's wait for the discussion result to do make any further action :)

Reply:
I still think it good to change.
@pvnr0082t @maruilian11 @PeterDaveHello Could you help us? Don't know which is better, `**/*` or `**/!(*[0-9])*`. 
Thank you.

Reply:
The naming scheme is correct in the latest version. I think it's fine to use `**/*`.

Reply:
@extend1994 would you help this PR?